brjs-sdk/sdk/libs/javascript/br-presenter/test-unit/src-test/br/presenter/testing/node/RootPresentationNode.js

```diff
@@ -6,6 +6,7 @@ var WritableProperty = require("br/presenter/property/WritableProperty");
 var RootPresentationNode = function()
 {
 	this.m_oPrivateProperty1 = new WritableProperty();
+	this._oAnotherPrivateProperty1 = new WritableProperty();
```

Comment:
@andyberry88 you've added this new property but don't seem to be using it in any of the tests